tests/Dockerfile

```diff
@@ -47,16 +45,17 @@ RUN dnf -y update && \
         tar \
         virt-install \
         zanata-client && \
-        dnf -y builddep /tmp/cockpit.spec && dnf clean all
-RUN npm -g install phantomjs-prebuilt
+    npm -g install phantomjs-prebuilt && \
+    curl -s -o /tmp/cockpit.spec https://raw.githubusercontent.com/cockpit-project/cockpit/master/tools/cockpit.spec && \
+    dnf -y builddep /tmp/cockpit.spec && dnf clean all && \
+    mkdir -p /usr/local/bin /home/user /build /secrets /cache/images /cache/github /build/libvirt/qemu
```

Comment:
The `mkdir` should move into the "quick & cheap" second RUN command below, while we are at it.